eventnotifier/slack/slack.go

```diff
@@ -44,9 +44,9 @@ type slackMessage struct {
 }
 
 type NotifierSlackConfig struct {
-	Enabled              bool
-	TokenFilepath        string
-	MirrorSlackChannelID string // To remove when we stop testing observer feature
+	Enabled              bool   `yaml:"enabled"`
+	TokenFilepath        string `yaml:"tokenFilepath"`
+	MirrorSlackChannelID string `yaml:"mirrorSlackChannelId"` // To remove when we stop testing observer feature
```

Comment:
Can we consider this field as deprecated if it will be removed in the future? If yes, can you mark it as deprecated?
```suggestion
        // Deprecated: will be removed when the observer feature is done 
	MirrorSlackChannelID string `yaml:"mirrorSlackChannelId"` // To remove when we stop testing observer feature
```